tests/virtual_machines_test.go

```diff
@@ -403,59 +404,32 @@ var _ = Describe("Virtual Machines", func() {
 		})
 		//2633
 		Context("When we re-apply a failed VM yaml", func() {
-			It("should allow to assign to the VM the same MAC addresses, with name as requested before and do not return an error", func() {
+			table.DescribeTable("should successfully allow to assign the same MAC addresses to the VM", func(firstVmName string, secondVmName string) {
 				err := initKubemacpoolParams(rangeStart, rangeEnd)
 				Expect(err).ToNot(HaveOccurred())
 
+				By("Applying not valid VM yaml with missing interface ")
 				vm1 := CreateVmObject(TestNamespace, false,
 					[]kubevirtv1.Interface{newInterface("br1", "02:00:ff:ff:ff:ff")},
 					[]kubevirtv1.Network{newNetwork("br1"), newNetwork("br2")})
+				vm1.Name = firstVmName
 
-				baseVM := vm1.DeepCopy()
+				vm2 := vm1.DeepCopy()
+				vm2.Name = secondVmName
 
 				err = testClient.VirtClient.Create(context.TODO(), vm1)
-				Expect(err).To(HaveOccurred())
-				Expect(strings.Contains(err.Error(), "every network must be mapped to an interface")).To(Equal(true))
-
-				baseVM.Spec.Template.Spec.Domain.Devices.Interfaces = append(baseVM.Spec.Template.Spec.Domain.Devices.Interfaces, newInterface("br2", ""))
+				Expect(err).To(HaveOccurred(), "should fail due to absence of a correspond interface to each network")
```

Comment:
I think we should keep the Expect message to make sure the err we get is the one we expect to get (and not something else like "network name is invalid" or else..)

Reply:
retracting this request